owslib/coverage/wcs100.py

```diff
@@ -126,8 +126,7 @@ def getCoverage(self, identifier=None, bbox=None, time=None, format=None, crs=No
         except StopIteration:
             base_url = self.url
 
-        if log.isEnabledFor(logging.DEBUG):
-            log.debug('WCS 1.0.0 DEBUG: base url of server: %s' % base_url)
+        log.debug('WCS 1.0.0 DEBUG: base url of server: %s' % base_url)
```

Comment:
Should be:
`log.debug('WCS 1.0.0 DEBUG: base url of server: %s', base_url)`
=> Don't interpret the template if not needed...